### DISCUSSION TITLE
artificial horizon location on osd moveable?

### DISCUSSION THREAD

**Discussion by P-I-Engineer** [iNavFlight/inav#7089] [Q&A] [answered]
i'd like to bias my artificial horizon lower since with a quad i tilt forward and my horizon bar disappears.

can't do it on the gui. Any thoughts?

**Comment by breadoven** [comment]
I think this is discussing the same thing which has a solution using the `osd_ahi_camera_uptilt_comp `setting in V 3.0.0:

https://github.com/iNavFlight/inav/issues/6990

**Comment by avsaase** [comment, marked as answer]
In 3.0, use
```
set osd_camera_uptilt = [uptilt angle]
set osd_ahi_camera_uptilt_comp = ON
```

**Comment by P-I-Engineer** [comment]
thanks, i'll try out the up tilt.
